.github/ISSUE_TEMPLATE/opengrant.md

```diff
@@ -6,6 +6,8 @@
 
 **Proposer:** `replace with your GitHub username`
 
+**Administrator:** `replace with pre-selected admin for targeted grants`
```

Comment:
Most applicants won't know what an Administrator is. Can we change the wording so this feels more optional, and doesn't scare away people who are submitting a non-sponsored open grant?
Maybe this:

**(Optional) Technical Sponsor:** `if you have previously discussed this project with a member of the IPFS or Filecoin project teams, include their name and/or github handle here`

Reply:
Awesome, thanks Mosh